Terminal.Gui/Terminal.Gui.csproj

```diff
@@ -59,11 +59,6 @@
     <PackageReference Include="System.IO.Abstractions" Version="21.0.22" />
     <PackageReference Include="System.Text.Json" Version="8.0.4" />
     <PackageReference Include="Wcwidth" Version="2.0.0" />
-    <ProjectReference Include="..\Analyzers\Terminal.Gui.Analyzers.Internal\Terminal.Gui.Analyzers.Internal.csproj">
-      <PrivateAssets>all</PrivateAssets>
-      <OutputItemType>Analyzer</OutputItemType>
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-    </ProjectReference>
```

Comment:
You removed it here in #3600 but it's again on the` v2_develop`.

https://github.com/gui-cs/Terminal.Gui/blob/7dc12a4c5844c1446df6ff2b337481ac89e5e609/Terminal.Gui/Terminal.Gui.csproj#L62-L66

Reply:
I removed the analyzer project and its tests, yes.

Didn't remove the actual TG UnitTests project.

But good thought on reformatting and whatnot. I'll add what's necessary to make ReSharper ignore those files as part of #3558 too, so it doesn't change things.